### PR TITLE
Sprint 1

### PR DESCRIPTION
**Pull request by fatjosephina**
Finished the four goals for the week.

### CODE REVIEW

File: Pully Penelope/Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        animator = GetComponent<Animator>();
+    }
+
+    private void Update()
+    {
+        positionChange = Vector3.zero;
+        positionChange.x = Input.GetAxisRaw("Horizontal");
+        positionChange.y = Input.GetAxisRaw("Vertical");
+
+        /*if (positionChange != Vector3.zero)
```
Comment (Akaiju) [inline Pully Penelope/Assets/Scripts/PlayerMovement.cs:26]: Why is this here?

**Review by Akaiju — APPROVED** [review]
Looks solid.

**Comment by Akaiju** [comment]
Style guide has been upheld, only thing I would add is documentation of what various parts do

File: Pully Penelope/Assets/Scripts/PlayerMovement.cs (hunk 1 above)
Comment (fatjosephina) [reply Pully Penelope/Assets/Scripts/PlayerMovement.cs:26]: Ah just commenting out an alternative solution in case I need it again. I'll probably delete it by the next iteration.
